### PR TITLE
Add webviewDebuggingEnabled prop to Safari Debugging steps

### PR DESCRIPTION
**Pull request by kevinvangelder**

### CODE REVIEW

File: docs/Debugging.md [hunk 1]
```diff
@@ -51,10 +51,11 @@ It's possible to debug WebView contents in the iOS simulator or on a device usin
...
 #### Steps:
 
+1. Enable the `webViewDebuggingEnabled` prop on your WebView
 1. Open Safari Preferences -> "Advanced" tab -> enable checkbox "Show Develop menu in menu bar"
-2. Start app with React Native WebView in iOS simulator or iOS device
-3. Safari -> Develop -> [device name] -> [app name] -> [url - title]
-4. You can now debug the WebView contents just as you would on the web
+1. Start app with React Native WebView in iOS simulator or iOS device
+1. Safari -> Develop -> [device name] -> [app name] -> [url - title]
+1. You can now debug the WebView contents just as you would on the web
```
Comment (TheAlmightyBob) [inline docs/Debugging.md:58]: That's a lot of step 1s....

Comment (jamonholmgren) [reply docs/Debugging.md:58]: Ha ... Markdown converts them to an ordered list, so doing all 1's is generally a good idea.

**Review by jamonholmgren — APPROVED** [review]

File: docs/Debugging.md (hunk 1 above)
Comment (TheAlmightyBob) [reply docs/Debugging.md:58]: TIL!

**Comment by TheAlmightyBob** [comment]
Thank you @kevinvangelder (and to @jamonholmgren for explaining the numbering oddity)! I'm actually merging #3029 instead though, because 1) it was first, 2) it includes Android, 3) it includes links to related documentation.

**Pull request closed without merging**